docs/source/upgrade_notes.rst

```diff
@@ -3,6 +3,39 @@
 Upgrade Notes
 =============
 
+|st2| v2.9
+----------
+
+* |st2| timers used to be run as part of ``st2rulesengine`` process until versions older than ``v2.9``.
```

Comment:
Maybe reword the first couple sentences (everything is the same after ` ``st2timersengine`` is the new...`):
```
* |st2| timers moved from the ``st2rulesengine`` to the ``st2timersengine`` in ``v2.9``. Moving timers
  out of the rules engine allows scaling rules and timers independently. ``st2timersengine`` is the new
  process that schedules all the user timers. Please note that when upgrading from older versions, you
  will need to carefully accept changes to ``st2.conf`` file. Otherwise, you risk losing access to
  ``st2`` database in MongoDB.
```